base/logger_external.go

```diff
@@ -73,6 +74,29 @@ func (GoCBCoreLogger) Log(level gocbcore.LogLevel, offset int, format string, v
 	return nil
 }
 
+// ******************************************************
+// SG Replicate Logging
+// ******************************************************
+// Log levels are mapped as follows:
+//   Debug   -> SG Debug
+//   Normal  -> SG Info
+//   Warning -> SG Warn
+//   Error   -> SG Error
+//   Panic   -> no-op
+
+func sgreplicateLogFn(level clog.LogLevel, format string, args ...interface{}) {
+	switch level {
+	case clog.LevelDebug:
+		Debugf(KeyReplicate, format, args...)
+	case clog.LevelNormal:
+		Infof(KeyReplicate, format, args...)
+	case clog.LevelWarning:
+		Warnf(KeyReplicate, format, args...)
+	case clog.LevelError:
+		Errorf(KeyReplicate, format, args...)
```

Comment:
I think @adamcfraser 's comments from yesterday weren't posted - but as discussed, need to handle LevelPanic at least in an Error log.

Reply:
Ah yeah...